demo-transactions/src/main/resources/application.yaml

```diff
@@ -33,9 +33,13 @@ spring:
       generate_statistics: true
     database-platform: org.hibernate.dialect.MySQL8Dialect
   flyway:
-    enabled: true
+    enabled: false
```

Comment:
Because i created FlywayConfig file that creates 2 flyway migration schemas, and i can't use flyway enable since it overrides the other. With 2 flyway schemas we can have V0_0_0 in query and also V0_0_0 in command. So we complete seperate the 2 databases. i think this as also an ideia that patrik added on the last EDM event tell me what you think 🙈